code/modules/mob/living/carbon/xenomorph/castes/queen/abilities_queen.dm

```diff
@@ -458,6 +458,13 @@
 	selected_xeno.hud_set_queen_overwatch()
 	selected_xeno.handle_xeno_leader_pheromones(xeno_ruler)
 
+	// Xenos with specialized icons (Queen, King, Shrike) do not need to have their icon returned to normal
+	if(selected_xeno.xeno_caste.minimap_icon != "xeno")
```

Comment:
I wish I could instead check against the default /datum/xeno_caste.minimap_icon but I have no clue how.

Can anyone write it out so I dont have to hardcode a check against "xeno"?